CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```

Comment:
If you are about to make a release immediately after this PR, might as well change [Unreleased] to the version number and add a new empty unreleased section, and add it's link to the bottom of the file.

Reply:
Okay, I'll do that.